modulemd/v2/include/modulemd-2.0/modulemd-component.h

```diff
@@ -36,12 +36,27 @@ struct _ModulemdComponentClass
   void (*set_name) (ModulemdComponent *self, const gchar *name);
   const gchar *(*get_name) (ModulemdComponent *self);
   gboolean (*validate) (ModulemdComponent *self, GError **error);
+  gboolean (*equals) (ModulemdComponent *self_1, ModulemdComponent *self_2);
 
   /* Padding to allow adding up to 8 new virtual functions without
    * breaking ABI. */
   gpointer padding[8];
 };
 
+/**
+ * modulemd_component_equals:
+ * @self_1: A #ModulemdComponent object
+ * @self_2: A #ModulemdComponent object
+ *
+ * Returns: TRUE, if both the objects are equal. FALSE, otherwise
+ *
+ * Since: 2.2
```

Comment:
I missed this before, sorry. 2.2 has already been released, so new functions need to be `Since: 2.3` now. I will fix this while merging. No need to resubmit.